docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -213,6 +228,85 @@ To add a Task to the end of a Mission:
 
     :::
 
+### Custom Tasks  {#custom-task}
+
+Users can create their own tasks following a specific template and place them within the **/onav_log/custom_tasks** directory.
+Restarting the UGV or dockers will trigger the system to load the custom task that was created, making it available for mission use.

```

Comment:
Is 'dockers' the standard term we are using throughout the website? 

- dockers
- Dockers
- containers
- Containers

Reply:
I think we just remove the "or dockers" entirely here as most users won't be interacting at that level.